subprojects/gradle/runner/device-provider/src/main/kotlin/com/avito/android/runner/devices/DevicesProviderFactory.kt

```diff
@@ -1,13 +1,19 @@
 package com.avito.android.runner.devices
 
 import com.avito.android.runner.devices.model.DeviceType
+import com.avito.android.stats.SeriesName
+import java.io.File
 
 public interface DevicesProviderFactory {
 
     public fun create(
         deviceType: DeviceType,
+        projectName: String,
         configurationName: String,
+        tempLogcatDir: File,
+        outputDir: File,
         logcatTags: Collection<String>,
-        kubernetesNamespace: String
+        kubernetesNamespace: String,
+        runnerPrefix: SeriesName
     ): DevicesProvider
```

Comment:
And here. These new parameters look like implementation details. I'd like to hide them from the public interface.

Reply:
true, i'll definitely hide it on later steps